pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -628,44 +627,39 @@ void NpyArr_iterBegin(JSOBJ _obj, JSONTypeContext *tc) {
         obj = (PyArrayObject *)_obj;
     }
 
-    if (PyArray_SIZE(obj) < 0) {
```

Comment:
This branch of the condition was unreachable

Reply:
@shoyer is there some special case this might have been intended for?

Reply:
I'm not familiar with the context here, but I agree that this was unreachable.

Maybe it was intended to cover size 0 arrays, may not be representable in JSON? e.g., an shape with shape `(0, 1)`